content/millennium/r4/financial/general/charge-item.md

```diff
@@ -83,19 +83,22 @@ Search for ChargeItems that meet supplied query parameters:
 
 ### Parameters
 
- Name                  | Required?       | Type          | Description
------------------------|-----------------|---------------|-------------------------------------------------------
- `_id`                 | This or context | [`token`]     | The logical resource id associated with the ChargeItem.
- `context`             | This or _id     | [`token`]     | Encounter associated with event.
- `-status`             | See notes       | [`token`]     | The status of the ChargeItem. Example: `billable`
- [`_count`]            | No              | [`number`]    | The maximum number of results to return.
+ Name                  | Required?                  | Type          | Description
+-----------------------|----------------------------|---------------|-------------------------------------------------------
+ `_id`                 | This or context or account | [`token`]     | The logical resource id associated with the ChargeItem.
+ `context`             | This or _id or account     | [`reference`] | Encounter associated with event.
+ `account`             | This or context or _id     | [`reference`] | Account associated with the ChargeItem.
+ `-status`             | See notes                  | [`token`]     | The status of the ChargeItem. Example: `billable`
+ [`_count`]            | No                         | [`number`]    | The maximum number of results to return.
 
 Notes:
 
 - When searching by `_id`, only a single ChargeItem id can be provided.
 - `-status` must be provided with `context`.
+- `-status` must be provided with `account`.
```

Comment:
Let's combine these. i.e.
```
`-status` must be provided with `context` and `account`.
```

Reply:
Addressed in 2e4df3f24bdeb9538a6e6875435ea5f695bd59b5

Reply:
This commit doesn't have this change.

Reply:
24bcb410398f2d17801fb6aec8534be947458102